lib/v2/fxiaoke/radar.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    'fxiaoke.com': {
+        _name: '纷享销客 CRM',
+        '.': [
+            {
+                title: '全部文章 - 纷享销客 CRM',
```

Comment:
No need to repeat `_name` in title
```suggestion
                title: '全部文章',
```